sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -2181,28 +2181,34 @@ static const char *NoOptStr = "-ze-opt-disable";
 static const char *O1OptStr = "-ze-opt-level=1";
 static const char *O2OptStr = "-ze-opt-level=2";
 
-// Returns plugin specific backend optimization option.
-// Return '-ze-opt-disable' for opt_level = 0.
-// Return '-ze-opt-level=1' for opt_level = 1/2.
-// Return '-ze-opt-level=2' for opt_level = 3.
-pi_result piPluginGetBackendOptimizationOption(int opt_level,
-                                               const char **backend_option) {
-  switch (opt_level) {
-  case 0:
+// Returns plugin specific backend option.
+// Current support is only for optimization options.
+// Return '-ze-opt-disable' for frontend_option = -O0.
+// Return '-ze-opt-level=1' for frontend_option = -O1 or -O2.
+// Return '-ze-opt-level=2' for frontend_option = -O3.
+pi_result piPluginGetBackendOption(pi_platform platform,
+                                   const char *frontend_option,
+                                   const char **backend_option) {
+  if (frontend_option == nullptr) {
+    return PI_ERROR_INVALID_VALUE;
+  }
+  if (!strcmp(frontend_option, "")) {
```

Comment:
Please use `std::string_view`s non-member `operator==` (https://en.cppreference.com/w/cpp/string/basic_string_view/operator_cmp) instead: https://godbolt.org/z/jzq8szWMK

Reply:
This is a nice suggestion. Let me try. Thanks